### DISCUSSION TITLE
Create list of all images for preloading

### DISCUSSION THREAD

**Discussion by kittymimo** [jspsych/jsPsych#1153] [Q&A] [answered]
Hi,

I would like to preload my images using `jsPsych.init({preload_images: images})`. I have images that are listed in a separate stimuli file, and others that are not included in the stimuli file because they stay the same across all trials. I am able to create a list of all the images from my stimuli file using this script:

```
var images = [];
    for (var i=0; i<stimuli.length; i++) {
    	images.push(stimuli[i].image1);
    }
    for (var i=0; i<stimuli.length; i++) {
    	images.push(stimuli[i].image2);
    }
    for (var i=0; i<stimuli.length; i++) {
    	images.push(stimuli[i].bonne_reponse);
    }
```
I am wondering how I can add the images that are not included in my stimuli file so that I can preload them too. 

Thanks!

Catherine

**Comment by jodeleeuw** [comment, marked as answer]
Hi Catherine,

I'm not sure if I understand exactly what you want to do, but you could certainly add other images with `images.push()`. You can just list them like this:

`images.push('image1.jpg', 'image2.jpg', 'image3.jpg')`

**Comment by kittymimo** [comment]
Hi! It looks like it's working, thank you!
